mautrix_telegram/license.py

```diff
@@ -29,7 +28,7 @@ def get_instance_id(log: logging.Logger = logging.getLogger()) -> str:
     if not _instance_id:
         license_file_path = (
             os.environ.get("MAUTRIX_TELEGRAM_LICENSE_PATH") or
-            str(Path(__file__).parents[1].joinpath("licenses", "instanceId"))
+            os.path.abspath(os.path.join("licenses", "instanceId"))
```

Comment:
Since `mautrix-telegram` is launched as a module (with `-m`) instead of as a file/binary to run, the default license file path can't safely be made relative to a module location, since there's no guarantee of how the running Python environment installed its packages.

This notably affects the container built by this repo's Dockerfile, which installs `mautrix_telegram` under `site-packages`, which is probably somewhere we don't want to be writing files to.

None of this applies to Node-based bridges, for which installation environments are more rigid and `__dirname` is a much more stable path than `__file__`.